### PR TITLE
Add missing BigInt type for value type query API

### PR DESCRIPTION
**Pull request by galpeter**
Extended the `jerry_type_t` enum with `JERRY_TYPE_BIGINT` and added it to the `jerry_value_get_type`.

### CODE REVIEW

File: tests/unit-core/test-api-value-type.c [hunk 1]
```diff
@@ -99,6 +99,31 @@ main (void)
...
+
+    /* Check bigint wrapped in object type */
+    jerry_char_t object_bigint_src[] = "Object(5n)";
+    jerry_value_t object_bigint = jerry_eval (object_bigint_src, sizeof (object_bigint_src) - 1, JERRY_PARSE_NO_OPTS);
+    TEST_ASSERT (!jerry_value_is_error (object_bigint));
+
+    jerry_type_t object_type_info = jerry_value_get_type (object_bigint);
+
+    TEST_ASSERT (object_type_info != JERRY_TYPE_NONE);
+    TEST_ASSERT (object_type_info == JERRY_TYPE_OBJECT);
```
Comment (rerobika) [inline tests/unit-core/test-api-value-type.c:122]: Can you extend the `jerry_object_type_t` enum with bigint wrapped objects as well?

Comment (galpeter) [reply tests/unit-core/test-api-value-type.c:122]: That is already there (if I remember correctly).

Comment (rerobika) [reply tests/unit-core/test-api-value-type.c:122]: Indeed! Just ignore my comment.

**Review by rerobika — APPROVED** [review]
LGTM

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**